database/migrations/2018_04_07_031106_create_users_table.php

```diff
@@ -15,8 +15,11 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
-            $table->string('email')->unique();
-            $table->string('password', 60);
+            $table->integer('national_id')->unique();
+            $table->integer('phone_number')->unique();
+            $table->integer('password');
```

Comment:
Although the user will enter a 4 digit password, after hashing it will be a string in the db